### ISSUE TITLE
jinja[invalid] when using a file lookup on ansible-core 2.13.6

### ISSUE THREAD

**Issue by evgeni** [ansible/ansible-lint#2661]
##### Summary

When using the `file` lookup in a playbook while the actual file is not available during linting, the following error is raised on Ansible core 2.13.6:
```
jinja[invalid]: The 'file' lookup had an issue accessing the file '/tmp/nonexistant'
```

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
% ansible --version                 
ansible [core 2.13.6]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/evgeni/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /tmp/tmp.8BwG0b1gaU/venv/lib64/python3.10/site-packages/ansible
  ansible collection location = /home/evgeni/.ansible/collections:/usr/share/ansible/collections
  executable location = /tmp/tmp.8BwG0b1gaU/venv/bin/ansible
  python version = 3.10.7 (main, Sep  7 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-1)]
  jinja version = 3.1.2
  libyaml = True

% ansible-lint --version
ansible-lint 6.8.6 using ansible 2.13.6
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

Fedora 36, but should not be relevant

##### STEPS TO REPRODUCE

example playbook (ignore the silliness of the task ;)) :
```yaml
---
- hosts: localhost
  name: Test
  tasks:
    - name: Simple task
      ansible.builtin.copy:
        dest: /tmp/something
        content: "{{ lookup('file', '/tmp/nonexistant', errors='ignore') }}"
        mode: '0644'
```

##### Desired Behavior

```console
% ansible-lint test.yml             
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: test.yml
[WARNING]: Unable to find '/tmp/nonexistant' in expected paths (use -vvvvv to see paths)

Passed with production profile: 0 failure(s), 0 warning(s) on 1 files.
```

##### Actual Behavior

```console
% ansible-lint test.yml 
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: test.yml
WARNING  Listing 1 violation(s) that are fatal
jinja[invalid]: The 'file' lookup had an issue accessing the file '/tmp/nonexistant'. file not found, use -vvvvv to see paths searched
test.yml:5 Task/Handler: Simple task

You can skip specific rules or tags by adding them to your configuration file:
# .config/ansible-lint.yml
warn_list:  # or 'skip_list' to silence them completely
  - jinja[invalid]  # Rule that looks inside jinja2 templates.

              Rule Violation Summary               
 count tag            profile rule associated tags 
     1 jinja[invalid] basic   formatting           

Failed after min profile: 1 failure(s), 0 warning(s) on 1 files.
```

##### Additional information

The same check works just fine with ansible-core 2.12.10 and 2.14.0 :confused: 

```
ansible [core 2.12.10]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/evgeni/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /tmp/tmp.8BwG0b1gaU/venv/lib64/python3.10/site-packages/ansible
  ansible collection location = /home/evgeni/.ansible/collections:/usr/share/ansible/collections
  executable location = /tmp/tmp.8BwG0b1gaU/venv/bin/ansible
  python version = 3.10.7 (main, Sep  7 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-1)]
  jinja version = 3.1.2
  libyaml = True

ansible [core 2.14.0]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/evgeni/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /tmp/tmp.8BwG0b1gaU/venv/lib64/python3.10/site-packages/ansible
  ansible collection location = /home/evgeni/.ansible/collections:/usr/share/ansible/collections
  executable location = /tmp/tmp.8BwG0b1gaU/venv/bin/ansible
  python version = 3.10.7 (main, Sep  7 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-1)] (/tmp/tmp.8BwG0b1gaU/venv/bin/python)
  jinja version = 3.1.2
  libyaml = True
```

**Comment by evgeni** [comment]
it gets better, this is fine on core 2.13.0-2.13.5, only 2.13.6 shows this behaviour

**Comment by evgeni** [comment]
(talking to myself, sorry)
seems the real bug is reported here: https://github.com/ansible/ansible/issues/79291

**Comment by peterge1998** [comment]
Hey, I feel like I have the same problem on Fedora 37.

```
$ ansible-lint --version
ansible-lint 6.9.1 using ansible 2.13.6
```
```
jinja[invalid]: The 'file' lookup had an issue accessing the file '../../shared_files/files/ansible_deployment_ssh_key.pub'. file not found, use -vvvvv to see paths searched (warning)

roles/common/tasks/main.yml:77 Task/Handler: [...]

              Rule Violation Summary               
 count tag            profile rule associated tags 
     1 jinja[invalid] basic   formatting (warning) 
```

When I am in `roles/common/tasks/`, a cat command on `../../shared_files/files/ansible_deployment_ssh_key.pub` works just fine...

**Comment by peterge1998** [comment]
> Hey, I feel like I have the same problem on Fedora 37.
> 
> ```
> $ ansible-lint --version
> ansible-lint 6.9.1 using ansible 2.13.6
> ```
> 
> ```
> jinja[invalid]: The 'file' lookup had an issue accessing the file '../../shared_files/files/ansible_deployment_ssh_key.pub'. file not found, use -vvvvv to see paths searched (warning)
> 
> roles/common/tasks/main.yml:77 Task/Handler: [...]
> 
>               Rule Violation Summary               
>  count tag            profile rule associated tags 
>      1 jinja[invalid] basic   formatting (warning) 
> ```
> 
> When I am in `roles/common/tasks/`, a cat command on `../../shared_files/files/ansible_deployment_ssh_key.pub` works just fine...

After upgrading ansible-lint to 6.10.0, the error doesn't show up anymore. Thank you guys <3
